var/spack/repos/builtin/packages/qt-base/package.py

```diff
@@ -160,6 +161,7 @@ class QtBase(QtPackage):
             "https://github.com/qt/qtbase/commit/c3d3e7312499189dde2ff9c0cb14bd608d6fd1cd.patch?full_index=1",
```

Comment:
Doesn't this patch need to be applied to 6.5.2 as well? https://github.com/qt/qtbase/commit/c3d3e7312499189dde2ff9c0cb14bd608d6fd1cd isn't in 6.5.2.